aquadoggo/src/db/stores/operation.rs

```diff
@@ -144,7 +144,7 @@ impl OperationStore<VerifiedOperation> for SqlStorage {
                             .bind(name.to_owned())
                             .bind(value.field_type().to_string())
                             .bind(db_value)
-                            .bind(index.to_string())
+                            .bind(index as i32)
```

Comment:
:middle_finger: PostgreSQL was complaining that this column is actually a number (`NUMERIC`).